packages/runtime/container-runtime/src/gc/garbageCollection.ts

```diff
@@ -1083,7 +1086,10 @@ export class GarbageCollector implements IGarbageCollector {
 	 * before runnint GC next.
 	 */
 	private triggerAutoRecovery(nodePath: string) {
-		if (this.mc.config.getBoolean(disableAutoRecoveryKey) === true) {
+		if (
+			!this.configs.tombstoneAutorecoveryEnabled ||
+			this.mc.config.getBoolean(disableAutoRecoveryKey) === true
```

Comment:
Can we remove this config? Doesn't have to be in this PR.

Reply:
Yeah it can be cleaned up later (maybe in your PR)